packages/uniforms-antd/src/BoolField.js

```diff
@@ -6,8 +6,8 @@ import filterDOMProps from 'uniforms/filterDOMProps';
 
 import wrapField from './wrapField';
 
-const Bool = props =>
-    wrapField(props, (
+const Bool = ({labelCol, wrapperCol, ...props}) =>
```

Comment:
Right, i beleive `filterDOMProps.register` is cleaner solution here. Thanks that you point them to me.